NEWS.md

```diff
@@ -1,5 +1,7 @@
 # httr 1.2.1.9000
 
+* Added `pause_min` to `RETRY` requests, allowing for sub-second
```

Comment:
Certainly, sorry I missed that. Since I have no "issue" number, would you prefer just `(@r2evans)` or `(@r2evans !461)`?